tests/semantic_analyser.cpp

```diff
@@ -787,9 +787,6 @@ stdin:1:12-21: ERROR: lhist() should be directly assigned to a map
 kprobe:f { @[lhist()] = 1; }
            ~~~~~~~~~
 stdin:1:12-21: ERROR: lhist() requires 4 arguments (0 provided)
-kprobe:f { @[lhist()] = 1; }
-           ~~~~~~~~~
-stdin:1:12-21: ERROR: lhist_t cannot be used as a map key
```

Comment:
this error is not needed as the one above it "lhist() should be directly assigned to a map" covers it